Website/web.config

```diff
@@ -159,7 +159,7 @@
       </expressionBuilders>
     </compilation>
     <!-- permits errors to be displayed for remote clients -->
-    <customErrors mode="off"/>
+    <customErrors mode="RemoteOnly"/>
```

Comment:
In the future, you should ignore any development updates like this.  You can make the change to suit your needs to test, but we shouldn't be committing these kinds of updates in a PR.  :)